ItalianTranslation_PlanetCoaster.xml

```diff
@@ -4121,183 +4121,183 @@
+		<entry key="BuildingPartDesc_FAD_PipShot_Juice_03" translation="Goditi tutti i gusti della natura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Juice_04" translation="Goditi tutti i gusti della natura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Juice_05" translation="Goditi tutti i gusti della natura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Juice_06" translation="Goditi tutti i gusti della natura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Juice_07" translation="Goditi tutti i gusti della natura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Juice_08" translation="Goditi tutti i gusti della natura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Juice_10" translation="Goditi tutti i gusti della natura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Smoothie_02" translation="Come natura voleva. Frullato.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Smoothie_03" translation="Come natura voleva. Frullato.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Smoothie_04" translation="Come natura voleva. Frullato.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Smoothie_05" translation="Come natura voleva. Frullato.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Smoothie_06" translation="Come natura voleva. Frullato.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Smoothie_07" translation="Come natura voleva. Frullato.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Smoothie_08" translation="Come natura voleva. Frullato.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Smoothie_10" translation="Come natura voleva. Frullato.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Water_02" translation="Incredibilmente Pura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Water_03" translation="Incredibilmente Pura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Water_04" translation="Incredibilmente Pura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Water_05" translation="Incredibilmente Pura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Water_06" translation="Incredibilmente Pura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Water_07" translation="Incredibilmente Pura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Water_08" translation="Incredibilmente Pura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PipShot_Water_10" translation="Incredibilmente Pura.&#10;Set Pipshot"/>
+		<entry key="BuildingPartDesc_FAD_PizzaGenericSlice" translation="Alimenti di plastica senza marchio."/>
+		<entry key="BuildingPartDesc_FAD_PizzaPenBadge" translation="Mettere il sapore al suo posto.&#10;Set Capitan Manzo"/>
```

Comment:
Sei sicuro che sia così? Mi sembra poco sensato

Reply:
"Putting flavour back where it belongs".. -back where it belongs- significa -al suo posto-..
Se preferisci ne faccio una più libera.. come ti dicevo le espressioni inglesi non sono molto orecchiabili

Reply:
Se riuscissi a renderla più orecchiabile sarebbe fantastico

Reply:
Tipo: "I sapori di una volta"? Mi sembra carino

Reply:
Direi che è fantastico